docs/standard/threading/threading-objects-and-features.md

```diff
@@ -16,16 +16,16 @@ Along with the <xref:System.Threading.Thread?displayProperty=nameWithType> class
 |Title|Description|  
 |-----------|-----------------|  
 |[The managed thread pool](the-managed-thread-pool.md)|Describes the <xref:System.Threading.ThreadPool?displayProperty=nameWithType> class, which provides a pool of worker threads that are managed by .NET.|  
-|[Timers](timers.md)|Describes timers that can be used in a multithreaded environment.|
-|[Overview of synchronization primitives](overview-of-synchronization-primitives.md)|Describes classes that can be used to synchronize access to data or control thread interaction.|
+|[Timers](timers.md)|Describes .NET timers that can be used in a multithreaded environment.|
+|[Overview of synchronization primitives](overview-of-synchronization-primitives.md)|Describes types that can be used to synchronize access to a shared resource or control thread interaction.|
 |[EventWaitHandle, AutoResetEvent, CountdownEvent, ManualResetEvent](eventwaithandle-autoresetevent-countdownevent-manualresetevent.md)|Describes managed event wait handles, which are used to synchronize thread activities by signaling and waiting for signals.|
-|[Mutexes](mutexes.md)|Describes how to use a <xref:System.Threading.Mutex?displayProperty=nameWithType> to synchronize access to an object or to build your own synchronization mechanisms.|
+|[Mutexes](mutexes.md)|Describes <xref:System.Threading.Mutex?displayProperty=nameWithType>, which grants an exclusive access to a shared resource.|
 |[Interlocked operations](interlocked-operations.md)|Describes the <xref:System.Threading.Interlocked?displayProperty=nameWithType> class, which provides atomic operations for variables that are shared by multiple threads.|
-|[Reader-Writer Locks](reader-writer-locks.md)|Describes the <xref:System.Threading.ReaderWriterLockSlim?displayProperty=nameWithType> class, which provides single-writer/multiple-reader semantics.|
-|[Semaphore and SemaphoreSlim](semaphore-and-semaphoreslim.md)|Describes the <xref:System.Threading.Semaphore?displayProperty=nameWithType> class and explains how to use it to control access to limited resources.|
+|[Reader-Writer Locks](reader-writer-locks.md)|Describes the <xref:System.Threading.ReaderWriterLockSlim?displayProperty=nameWithType> class, which provides single-writer/multiple-reader access to a shared resource.|
+|[Semaphore and SemaphoreSlim](semaphore-and-semaphoreslim.md)|Describes the <xref:System.Threading.Semaphore?displayProperty=nameWithType> class, which limits number of threads that can access a shared resource concurrently.|
```

Comment:
I'm not sure that this description represents the typical semaphore usage. I'm inclined to say that it's sort of the other way around - there's a limited number of resources (N.B not a single resource) and an arbitrary number of threads that try to use those resources.

Reply:
Trying to find a middle ground: mention "a pool of resources" as in the API docs:
https://docs.microsoft.com/en-us/dotnet/api/system.threading.semaphore?view=netframework-4.7.2